src/marqo/tensor_search/tensor_search.py

```diff
@@ -1003,8 +1067,23 @@ def _vector_text_search(
         # empty body means that there are no vector fields associated with the index.
         # This probably means the index is emtpy
         return {"hits": []}
+    
+    end_preprocess_time = timer()
+    total_preprocess_time = end_preprocess_time - start_preprocess_time
+    logger.info(f"search (tensor) pre-processing: took {(total_preprocess_time):.3f}s to vectorize and process query.")
+
+    # SEARCH TIMER-LOGGER (roundtrip)
+    start_search_http_time = timer()
     response = HttpRequests(config).get(path=F"{index_name}/_msearch", body=utils.dicts_to_jsonl(body))
 
+    end_search_http_time = timer()
+    total_search_http_time = end_search_http_time - start_search_http_time
+    
+    num_responses = len(response["responses"])     # TODO JOSHUA find num_results
+    logger.info(f"search (tensor) roundtrip: took {(total_search_http_time):.3f}s to send search query (roundtrip) to Marqo-os and received {num_responses} responses.")
+
```

Comment:
Can we get the took time from the Marqo-os response too? In the same way we had "Marqo-os index" timing